L1Trigger/Phase2L1ParticleFlow/python/L1BJetProducer_cff.py

```diff
@@ -0,0 +1,21 @@
+import FWCore.ParameterSet.Config as cms
+
+from L1Trigger.Phase2L1ParticleFlow.l1pfJetMet_cff import L1TPFJetsExtendedTask
+
+from L1Trigger.Phase2L1ParticleFlow.L1BJetProducer_cfi import L1BJetProducer as _L1BJetProducer
```

Comment:
Same comment as above.

Reply:
L1Trigger/Phase2L1ParticleFlow/python/L1BJetProducer_cfi.py is a file with identical name and content than the one created by the fillDescriptions method, and as such it can only generate confusion, for example when parsing from the release. I agree with @AdrianoDee : please remove this explicit file

Reply:
Here there is nothing to remove: it is a cff which clones from other configurations and it must stay

Reply:
In L1BJetProducer_cff.py there is nothing to remove, instead: it is a cff which clones from other configurations and it must stay

Reply:
Okay. I can remove it. I'm finalizing the other requests. Should be ready soon~ish.